ecs-init/config/common.go

```diff
@@ -117,11 +122,14 @@ func DockerUnixSocket() (string, bool) {
 
 // getBaseLocationForRegion fetches the bucket URI from list of S3 Buckets by region name or default if key is not found
 func getBaseLocationForRegion(regionName string) string {
+	if regionName == RegionNameNotFound {
```

Comment:
@samuelkarp can you confirm [this mocked response in TestDownloadAgentSuccessWithRegionFailure](https://github.com/aws/amazon-ecs-init/pull/87/files#diff-0be318ad582dbc3fdf05dca3c0e463bfR418) adequately models the failure you've mentioned?